src/transformers/models/gpt_neox/modeling_gpt_neox.py

```diff
@@ -873,3 +878,86 @@ def forward(
             hidden_states=outputs.hidden_states,
             attentions=outputs.attentions,
         )
+
+
+class GPTNeoXForTokenClassification(GPTNeoXPreTrainedModel):
+    def __init__(self, config):
+        super().__init__(config)
+        self.num_labels = config.num_labels
+
+        self.gpt_neox = GPTNeoXModel(config)
+        if hasattr(config, "classifier_dropout") and config.classifier_dropout is not None:
+            classifier_dropout = config.classifier_dropout
+        elif hasattr(config, "hidden_dropout") and config.hidden_dropout is not None:
+            classifier_dropout = config.hidden_dropout
+        else:
+            classifier_dropout = 0.1
+        self.dropout = nn.Dropout(classifier_dropout)
```

Comment:
```suggestion
        self.dropout = nn.Dropout(config.classifier_dropout)
```
There is a config parameter that always has a default now, so let's use it.

Reply:
Haha I said the same glad to have a validation!

Reply:
Yes, I added the config parameter thanks to @ArthurZucker's feedback 👍